packages/compose/README.md

```diff
@@ -161,6 +161,39 @@ _Related_
 Given a component returns the enhanced component augmented with a component
 only re-rendering when its props/state change
 
+### throttle
+
+A simplified and properly typed version of lodash's `throttle`, that
+always uses timers instead of sometimes using rAF.
+
+Creates a throttled function that only invokes `func` at most once per
+every `wait` milliseconds (or once per browser frame). The throttled function
```

Comment:
Similarly to the `debounce` PR, we should remove the mention of browser frames here. Looks like I missed it in the original code.

Reply:
Indeed, addressed in https://github.com/WordPress/gutenberg/pull/44225/commits/4ccb6206f475c69f6acb52f509e8e17210074ce3